solr/core/src/java/org/apache/solr/core/SolrXmlConfig.java

```diff
@@ -568,4 +568,17 @@ private static PluginInfo getTracerPluginInfo(XmlConfigFile config) {
     Node node = config.getNode("solr/tracerConfig", false);
     return (node == null) ? null : new PluginInfo(node, "tracerConfig", false, true);
   }
+  
+  private static SharedStoreConfig loadSharedStoreConfig(NamedList<Object> nl) {
```

Comment:
fyi I moved the actual initialization method from SharedStoreConfig to SolrXmlConfig to avoid duplicating the existing field extraction methods or changing their visibility and so it aligns with how the rest of the section objects are being created. I've kept the SharedStoreConfig since I anticipate we'll start using it once we complete the enabling configurable's work.